src/Providers/Maps/Google/Features/Zoom.ts

```diff
@@ -30,6 +30,29 @@ namespace Provider.Maps.Google.Feature {
             );
         }
 
+        private _setBounds(useShapes: boolean) {
+            const bounds = new google.maps.LatLngBounds();
+            this._map.markers.forEach(function (item) {
+                if (item.provider === undefined) return;
+                const loc = item.provider.position.toJSON();
```

Comment:
I think we could remove the const and use the method directly

Reply:
Fixed!